languages/README.md

```diff
@@ -0,0 +1,134 @@
+
+A translator can start translating by following [the translation instructions](#how-to-translate).
+Translators should follow the [translation style guide](#translation-style-guide).
+
+### Reviewers
+
+Anyone, who builds [a new language structure](#how-to-add-a-new-language), will be the first reviewer
+of that language.
+
+Reviewers should track translation state of each document by editing 
+`languages/{language-code}/translations.md` file.
+
+There are 4 types of status:
+- Ready to start: any document is ready to be translated stay in this state.
+- Started: indicating someone is translating this document
+- PR reviewing: a translator finished the translation, and the work is being reviewed.
+- PR merged: the translation is successfully merged into the spec.
+
+Initially, the last edit time will be the time when the translation is merged.
+Reviewers should ensure translations are up-to-date with the English version as it changes, meanwhile 
+update the last edit time as well. 
+
+The following rules govern adding and removing reviewers:
+
+- Reviewers can be proposed via a PR to edit the `languages/{language-code}/reviewer.md` file.
+
+- Reviewers can be removed via a PR to edit the `languages/{language-code}/reviewer.md` file.
+
+## How to add a new language 
+
+This section is mainly for anyone who needs to build a new language structure.
+
+Firstly, check the [language list](languages.md) to see if the language
+you'd like to translate into already exists. If it doesn't exist, 
+and there is not a PR indicating someone is currently working on it,
+then you can start to add a new one.
+
+### Step 1. Create a PR to edit the languages.md
+
+Before starting doing any actual works, you should create a PR to edit the
+[language list](languages.md), adding a new language item to avoid redundant works from others.
+
+### Step 2. Build the structure locally
+
+After you cloned the repo to your local environment, you should create a directory 
+under `languages` directory, named after the specific 
+[language code](http://www.lingoes.net/en/translator/langcode.htm).
+
+Then, copy all original documents except `languages` directory to `languages/{language-code}`.
+
+*Note*: there is no need to copy project-related files(i.e.,LICENSE, OWNERS).
+
+For example:
+```
+copy cloudevents/spec.md to languages/zh-CN/cloudevents/spec.md
+copy README.md to languages/zh-CN/README.md
+......
+```
+You should also clear English content of copied documents to indicate 
+that the document is ready to be translated.
+
+The [reviewer list](zh-CN/reviewers.md) and the [translation list](zh-CN/translations.md)
+should be copied to `languages/{language-code}`.
+
+### Step 3. Create a new PR to upload the new structure
+
+Once you finished build the structure locally, you can create a PR to upload the changes.
+
+## How to translate
+
+### Step 1. Choose a document to translate 
+
+A translator can find a document to translate by filtering issues based on 
```

Comment:
when are these issues opened? who opens them? This is the first mention of "issues" in this doc.
I wonder if it would be easier for them to look at the existing languages/{language-code}/translations.md files and see which ones are available

Reply:
I used to think about having an admin to open issues. Let's say if we find it would be better to have a Chinese(or any other language) translation during a meeting. Then any volunteered admin can start open issues to publish translation task.
But it seems to be a lot works for admins, I'm not sure if this's a good way for others. I'd be happy to do such kind of works.
Or we can just let translators check `translations.md`, but they might see some outdated states of files.